test/multilocale/strings/remoteStringCast.future

```diff
@@ -0,0 +1,8 @@
+bug: string:string cast within on-clause gives follower error
+
+While this pattern isn't particularly interesting or useful as written
+here, it seems that it ought to be legal, yet it gives an error:
+"follower iterators accepting a non-POD argument by in-intent are not
+implemented".  The pattern is more interesting/useful in the context
+of a generic function where you want to cast an incoming argument to
+'x' regardless of its type and want it to also work with strings.
```

Comment:
Add a proper newline at the end?

Aside: I may be a good person to take a look at it, as I was the one who added this error to the compiler.